proposals/lip-0045.md

```diff
@@ -741,6 +745,118 @@ def isLive(chainID: ChainID) -> bool:
         return True
 ```
 
+#### send
+
+The `sendInternal` function is used to create and add a message to the outbox of a partner chain.
+
+```python
+def sendInternal(sendingAddress: Address,
+    module: Module,
+    crossChainCommand: CrossChainCommand,
+    receivingChainID: ChainID,
+    fee: uint64,
+    status: uint32,
+    params: object
+) -> None:
+
+    ccm = {
+        "nonce": ownChainAccount.nonce,
+        "module": module,
+        "crossChainCommand": crossChainCommand,
+        "sendingChainID": ownChainAccount.chainID,
+        "receivingChainID": receivingChainID,
+        "fee": fee,
+        "status": status,
+        "params": EMPTY_BYTES
+    }
+
+    paramsSchema = JSON schema associated with (module, crossChainCommand)
```

Comment:
I would move this also into the try so that an event is emitted if the schema does not exist.

Reply:
done